gradle/android-dbflow/app/src/main/java/mobi/porquenao/poc/kotlin/ui/MainActivity.kt

```diff
@@ -0,0 +1,37 @@
+package mobi.porquenao.poc.kotlin.ui
+
+import android.os.Bundle
+import android.support.v7.widget.LinearLayoutManager
+import android.support.v7.widget.RecyclerView
+import android.view.Menu
+import android.view.MenuItem
+import mobi.porquenao.poc.kotlin.R
+
+public class MainActivity : BaseActivity() {
+
+    lateinit var mAdapter: MainAdapter
+    lateinit var vList: RecyclerView
+
+    override fun onCreate(savedInstanceState: Bundle?) {
+        super.onCreate(savedInstanceState)
+        setContentView(R.layout.main)
+        vList = findViewById(R.id.list) as RecyclerView
```

Comment:
I'd suggest to add a comment that extension plugin must be installed extensions here. It is important to promote the greatness of our technology, and not send a mixed message: Kotlin is great with Android, but we won't show you that :)